bin/integration_tests

```diff
@@ -27,15 +27,14 @@ echo "Building and starting test env..."
 docker-compose build test-python
 docker-compose up -d test-python
 
-rm -rf $CURRENT_DIR/output/*
+rm -rf $CURRENT_DIR/out/*
 
 if [[ "$DEBUG" == "true" ]]; then
   docker-compose exec test-python bash
   exit
 fi
 
 docker-compose run \
-  --rm \
   --no-deps \
   test-python \
-  nose2 -v -s test/python/
```

Comment:
Can we add a .gitignore entry for the junit output?

Reply:
I believe it is already in the [.gitignore](https://github.com/cyberark/conjur-openapi-spec/pull/50/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R9)

Reply:
Ah great, thanks!